src/app/Plans/components/Wizard/GeneralForm.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { Form, FormGroup, TextArea, TextInput, Title } from '@patternfly/react-core';
+import SimpleSelect, { OptionWithValue } from '@app/common/components/SimpleSelect';
+import { Provider } from '@app/Providers/types';
+
+interface IGeneralFormComponent {
+  srcProviders: Provider[];
+  tgtProviders: Provider[];
```

Comment:
We should change "tgt" to "dest". After discussing with Mike, we have come to the consensus that "src" and "dest" work best for code legibility going forward. 

NOTE: The markup for ui fields will continue to read "Source" and "Target"

Reply:
@ibolton336, no worries I'll change it.